build.sbt

```diff
@@ -916,10 +720,15 @@ lazy val docs = project
     scalacOptions += "-Xlog-implicits",
     libraryDependencies ++= Seq("dev.zio" %% "zio" % Version.zio),
     projectName    := "ZIO Quill",
-    mainModuleName := (`quill-core-jvm` / moduleName).value,
-//    ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(
-//      `quill-engine-jvm`,
-//    ),
+    mainModuleName := (`quill-core` / moduleName).value,
+    // With Scala 2.12, these projects doc isn't compiling.
+    ScalaUnidoc / unidoc / unidocProjectFilter := inAnyProject -- inProjects(
+      `quill-engine`,
+      `quill-core`,
+      `quill-cassandra-monix`,
+      `quill-orientdb`,
+      `quill-doobie`,
+    ),
```

Comment:
Not sure to fully understand why I need to do this but that was the only way I found to make the publishing of the Scala 2.12 version to pass.

Without this, the ScalaDoc generation fails which makes the publishing process to fail